docs/file-formats/xml-data-file.md

```diff
@@ -997,22 +997,20 @@ They can have the following predefined properties:
 | `relatesTo`        | 0-n         | resptr-prop   |             |
 
 The segment bounds must be entered in seconds. 
-Decimal values for fractions of a second are allowed, e.g. `1.4:2`.
+Decimal values for fractions of a second are allowed, e.g. `1.4`.
 The segment from 0h 10min 00s to 0h 20min 00s would thus be represented by the interval `600:1200`.
```

Comment:
I would not show `600:1200` because that is not user.facing, but from what I could see happens during the xmlupload. I would only show user-facing formats in the documentation for example show the attributes `start="600" end="1200"` or the entire element.

Reply:
Good catch, I forgot to change that. Done with https://github.com/dasch-swiss/dsp-tools/commit/a0e279e23fb12af2286fe0b0b1066cad32ff9b21